src/components/device-page/index.tsx

```diff
@@ -57,7 +57,7 @@ export class DevicePage extends Component<DevicePageProps & GlobalState, {}> {
             <h3>{device.friendly_name}</h3>
             <ul className="nav nav-tabs">
```

Comment:
you only need change here, don't touch any other places.
Kindly follow this order:

About, Exposes, Bind, Reporting, State, Clusters